grr/server/grr_response_server/gui/ui/components/client_details/client_details.scss

```diff
@@ -2,12 +2,13 @@
 
 .details-container {
   display: flex;
-  // Set to full_screen_height - toolbar_height to enable div scrolling
-  height: calc(100% - 64px);
+  // Set to full_screen_height - toolbar_height - client_header_height to enable div scrolling
+  height: calc(100vh - 64px - 150px);
```

Comment:
I am not an expert in CSS languages, but cannot these values be put in some constants to avoid explaining the formula in the comment?